gradle/libs.versions.toml

```diff
@@ -47,20 +47,20 @@ kotlinxDatetime = "0.6.0"
 kotlinxColletionsImmutable = "0.3.7"
 #compose
 composeMultiplatform = "1.7.0-dev1743"
-composeBom = "2024.06.00"
+composeBom = "2024.08.00"
 composeLifecycleViewmodel = "2.8.0"
 composeNavigation = "2.8.0-alpha08"
 #other
 ktorServer = "3.0.0-beta-2"
 ktorClient = "3.0.0-beta-2"
 jetbrainsExposed = "0.52.0"
-logback = "1.4.2"
-ksp = "2.0.0-1.0.23"
+logback = "1.4.14"
+ksp = "2.0.10-1.0.24"
 detekt = "1.23.5"
-coil = "3.0.0-alpha07"
+coil = "3.0.0-alpha10"
 dependencyGuard = "0.4.3"
-androidxMedia3Exoplayer = "1.3.1"
-androidxMedia3Ui = "1.3.1"
+androidxMedia3Exoplayer = "1.4.1"
+androidxMedia3Ui = "1.4.1"
```

Comment:
Score: A

Reason:
- The version updates are well-organized and follow a consistent pattern.
- No issues with the structure or logic of the version updates.

Suggestion:
- None needed.